docs/start/start.md

```diff
@@ -5,12 +5,12 @@ disableTOC: true
 filterKey: integration
 ---
 
-The open-source Amplify Framework provides the following products to build fullstack iOS, Android, Web, and React Native apps:
-- **Amplify [CLI](../cli)** - Configure all the services needed to power your backend through a simple command line interface.
-- **Amplify [Libraries](../lib)** - Use case-centric client libraries to integrate your app code with a backend using declarative interfaces.
-- **Amplify UI components** - UI libraries for React, React Native, Angular, Ionic and Vue.
+The Amplify Framework consists of the following products, which can be used to build fullstack iOS, Android, Web, and React Native apps.
 
-The **Amplify [Console](https://aws.amazon.com/amplify/console/)** is an AWS service that provides a git-based workflow for continuous deployment & hosting of fullstack web apps. Cloud resources created by the Amplify CLI are also visible in the Amplify Console.
+- **Amplify [CLI](~/cli/cli.md)** - Configure all the services needed to power your backend through a simple command line interface.
+- **Amplify [Libraries](~/lib/lib.md)** - Use case-centric client libraries to integrate your app code with a backend using declarative interfaces.
+- **Amplify [UI components](~/ui/ui.md)** - UI libraries for React, React Native, Angular, Ionic and Vue.
+- **Amplify [Console](https://aws.amazon.com/amplify/console/)** is an AWS service that provides a git-based workflow for continuous deployment & hosting of fullstack web apps. Cloud resources created by the Amplify CLI are also visible in the Amplify Console.
```

Comment:
Should we replace "is an" with "-" to match the bullets about?

Reply:
Absolutely. Just pushed that change 👍